develop/TOOLS/NuGet/Consuming_NuGet.md

```diff
@@ -119,3 +119,39 @@ In case you are using a token that has an expiration date or if you entered the
 1. Select the credentials you want to edit and click *Edit* in the lower left corner.
 
 1. Update the username and/or password, and click *Save*.
+
+#### Issues with credentials
+
+In case you are having issues with the credentials in Visual Studio were each time the pop-up window for the credentials appear, you can try this way:
```

Comment:
If you encounter problems with credentials in Visual Studio, particularly when the credentials pop-up window appears repeatedly, you can follow these steps to resolve the issue: